tests/by-util/test_more.rs

```diff
@@ -1,6 +1,12 @@
 use crate::common::util::TestScenario;
 use is_terminal::IsTerminal;
 
+//Both following includes are only needed inside the test_more_invalid_file_perms()
+#[cfg(target_family = "unix")]
+use std::fs::{set_permissions, Permissions};
+#[cfg(target_family = "unix")]
+use std::os::unix::fs::PermissionsExt;
```

Comment:
I would move the includes inside the function so you don't need the comment and the attributes.